app/api/v1/currencies.py

```
@@ -0,0 +1,70 @@
+from app.core import settings
+from app.core.cache import cache_key_builder
+from app.db.dependency import get_db
+from app.db.model import Currencies
+from app.gw2.client import GW2Client
+
+router = APIRouter(prefix="/currencies", tags=["currencies"])
+
+
+@router.get("/", summary="Provides info about currencies", response_description="Currencies info")
+@cache(expire=settings.CACHE_TTL_SECONDS, namespace="account", key_builder=cache_key_builder)
+async def get_currencies(db: AsyncSession = Depends(get_db)):
```

Ruff (0.14.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)